src/components/Faculty/FeedbackFrom.tsx

```diff
@@ -0,0 +1,207 @@
+import {
+  Dialog,
+  DialogOverlay,
+  DialogContent,
+  DialogTrigger,
+  DialogClose,
+  DialogHeader,
+  DialogTitle,
+  DialogDescription,
+} from "../ui/dialog";
+import TextArea from "antd/es/input/TextArea";
+import { Button } from "@/components/ui/button";
+import { useState, useEffect } from "react";
+import { updateApproval } from "@/helpers/api";
+import Cookies from "js-cookie";
+import { useRouter } from "next/navigation";
+
+interface ButtonProps {
+  children?: React.ReactNode;
+  rows?: any[];
+  remarks?: string;
+  finalButton: boolean;
+}
+
+const UpdateApprovalPATCH = (rows: any[], remarks: string, status: string) => {
+  var allAppArray: any[] = [];
+  for (var i = 0; i < rows.length; i++) {
+    allAppArray.push({ id: rows[i].id, remarks, status });
+  }
+  updateApproval(Cookies.get("accessToken"), allAppArray);
+  window.location.reload();
+};
+
+const AcceptButton: React.FC<ButtonProps> = ({
+  rows = [],
+  children,
+  remarks = "",
+  finalButton,
+}) => {
+  return (
+    <Button
+      variant={"default"}
+      className="bg-sky-600 mx-4 hover:bg-blue-600"
+      onClick={() => {
+        if (finalButton) {
+          UpdateApprovalPATCH(rows, remarks, "APPROVED");
+        }
+      }}
+    >
+      Approve
+      <svg
```

Comment:
we might want to have svg in a component maybe and use it. the code might be cleaner. though your choice

Reply:
Ok